csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -56,26 +56,29 @@ public sealed class OutgoingRequestFrame : OutgoingFrame
         // _defaultBinaryContext is empty.
         private readonly bool _writeSlot0;
 
-        /// <summary>Create a new OutgoingRequestFrame for an operation with a single parameter.</summary>
+        /// <summary>Creates a new <see cref="OutgoingRequestFrame"/> for an operation with a single non-struct
+        /// parameter.</summary>
+        /// <typeparam name="T">The type of the operation's parameter.</typeparam>
```

Comment:
Should we use argument(s) instead of parameter?

Reply:
operation parameter is correct.